app/models.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date, datetime
+from uuid import UUID
+from pydantic import BaseModel, EmailStr , ValidationError,Field, field_validator 
+from pydantic_settings import BaseSettings
+
+
+
+# **************** Auth Models ***************
+class Token(BaseModel):
+    access_token: str
+    token_type: str
+
+class TokenData(BaseModel):
+    user_id: UUID | None = None
+
+
+class User(BaseModel):
+    username: str
+    email: EmailStr | None = None
+    is_active: bool = Field(default= True)
+    is_admin : bool = Field(default= False)
+    created_at: datetime | None = None
+    updated_at: datetime | None = None
+
+class UserInDB(User):
+    user_id : UUID
+    hashed_password: str
+
+
+class Settings(BaseSettings):
+    secret_key: str = Field(..., env="SECRET_KEY")
+    algorithm: str = Field("HS256")  
```

Comment:
If we have an environment variable called "algorithm" or "ALGORITHM", it will overwrite the value here. There is no need to add this value to the settings. Instead, define it as a variable in the auth file and make sure to reflect these changes across all the repo.

Reply:
I removed the algorithm from the environment variables and kept it in the settings for easier management.I will change it as you suggested.

Also, since I added the .env file to .gitignore, how should we share the secret key with the team?

Reply:
Thanks, no need to share the secret, because we run it locally. We build everything from scratch. We don't have a shared DB that everyone uses.

Reply:
Thanks for the clarification